biztalk/technical-guides/post-configuration-database-optimizations1.md

```diff
@@ -24,7 +24,7 @@ In addition to following the recommendations in [Pre-Configuration Database Opti
 
 - Database auto-growth should be set to a fixed number of megabytes instead of to a percentage (specify file growth **In Megabytes**). This should be done so if auto-growth occurs, it does so in a measured fashion which reduces the likelihood of excessive database growth. The growth increment should generally be no larger than 100 MB (for large files), 10 MB (for medium-sized files), or 1 MB (for small files).
 
-- When SQL Server increases the size of a file, the new space must first be initialized before it can be used. This is a blocking operation that involves filling the new space with empty pages. SQL Server 2005 running on Windows Server 2003 or later supports “instant file initialization.” This can greatly reduce the performance impact of a file growth operation. For more information, see "Database File Initialization" in the SQL Server 2008 documentation at [https://go.microsoft.com/fwlink/?LinkId=132063](/sql/relational-databases/databases/database-instant-file-initialization). This topic provides steps for enabling instant file initialization.
+- When SQL Server increases the size of a file, the new space must first be initialized before it can be used. This is a blocking operation that involves filling the new space with empty pages. SQL Server 2005 running on Windows Server 2003 or later supports “instant file initialization.” This can greatly reduce the performance impact of a file growth operation. For more information, see [Database File Initialization](https://go.microsoft.com/fwlink/?LinkId=132063) in the SQL Server 2008 documentation. This topic provides steps for enabling instant file initialization.
```

Comment:
```suggestion
- When SQL Server increases the size of a file, the new space must first be initialized before it can be used. This is a blocking operation that involves filling the new space with empty pages. SQL Server 2005 running on Windows Server 2003 or later supports “instant file initialization.” This can greatly reduce the performance impact of a file growth operation. For more information, see [SQL Server 2008 - Database File Initialization](/sql/relational-databases/databases/database-instant-file-initialization?redirectedfrom=MSDN&view=sql-server-ver16). This topic provides steps for enabling instant file initialization.
```